Classes/EventListener/AsyncEventListenerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace Neos\Cqrs\EventListener;
+
+/*
+ * This file is part of the Neos.Cqrs package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * Marker interface for asynchronous event listeners
+ */
+interface AsyncEventListenerInterface extends EventListenerInterface
+{
```

Comment:
Since you already use `getSynchronousListeners()` and `getAsynchronousListeners()`, this interface might better be named `AsynchronousEventListenerInterface`.